File: 3/src3/scores1.c
// Generates a bar chart of three scores

#include <cs50.h>
#include <stdio.h>

void chart(int score);

int main(void)
{
    // Get scores from user
    int score1 = get_int("Score 1: ");
    int score2 = get_int("Score 2: ");
    int score3 = get_int("Score 3: ");

    // Chart first score
    printf("Score 1: ");
    chart(score1);

    // Chart second score
    printf("Score 2: ");
    chart(score2);

    // Chart third score
    printf("Score 3: ");
    chart(score3);
}

// Generate bar
void chart(int score)
{
    // Output one hash per point
    for (int i = 0; i < score; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: 6/src6/weeks/2/scores1.py
# Generates a bar chart of three scores

from cs50 import get_int


def main():

    # Get scores from user
    score1 = get_int("Score 1: ")
    score2 = get_int("Score 2: ")
    score3 = get_int("Score 3: ")

    # Chart first score
    print("Score 1: ", end="")
    chart(score1)

    # Chart second score
    print("Score 2: ", end="")
    chart(score2)

    # Chart third score
    print("Score 3: ", end="")
    chart(score3)


def chart(score):

    # Output one hash per point
    for i in range(score):
        print("#", end="")
    print()


if __name__ == "__main__":
    main()
